### PR TITLE
resource/alicloud_kms_secret: Added the field policy, create_time; Improved alicloud_kms_secret testcase

### PR DESCRIPTION
**Pull request by MrWolong**

### CODE REVIEW

**Review by ChenHanZhang — COMMENTED** [review]
approved

File: alicloud/resource_alicloud_kms_secret_test.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func init() {
 func testSweepKmsSecret(region string) error {
```
Comment (MrWolong) [inline alicloud/resource_alicloud_kms_secret_test.go:32]: TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKmsSecret_basic0 -timeout=1200m

=== RUN   TestAccAliCloudKmsSecret_basic0

--- PASS: TestAccAliCloudKmsSecret_basic0 (32.11s)
=== RUN   TestAccAliCloudKmsSecret_basic0_twin

--- PASS: TestAccAliCloudKmsSecret_basic0_twin (7.76s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	39.972s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKmsSecret_basic1 -timeout=1200m

=== RUN   TestAccAliCloudKmsSecret_basic1

--- PASS: TestAccAliCloudKmsSecret_basic1 (733.76s)
=== RUN   TestAccAliCloudKmsSecret_basic1_twin

--- PASS: TestAccAliCloudKmsSecret_basic1_twin (644.16s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	1378.019s

TF_ACC=1 go test ./alicloud -v -run=TestAccAliCloudKmsSecret_basic2 -timeout=[120](https://github.com/aliyun/terraform-provider-alicloud/actions/runs/9253772323/job/25454356588?pr=7264#step:7:121)0m

=== RUN   TestAccAliCloudKmsSecret_basic2

--- PASS: TestAccAliCloudKmsSecret_basic2 (702.30s)
=== RUN   TestAccAliCloudKmsSecret_basic2_twin

--- PASS: TestAccAliCloudKmsSecret_basic2_twin (697.25s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	1399.662s

TF_ACC=1 go test ./alicloud -v -run=TestAccAlicloudKMSKey_basic -timeout=1200m

=== RUN   TestAccAlicloudKMSKey_basic
    provider_test.go:143: Skipping unsupported region eu-central-1. Supported regions: [cn-beijing cn-zhangjiakou cn-hangzhou cn-shanghai cn-shenzhen cn-hongkong ap-southeast-1 ap-southeast-2 ap-southeast-3 ap-southeast-5 us-east-1]. Using ap-southeast-1 as this test region

--- PASS: TestAccAlicloudKMSKey_basic (21.45s)
PASS
ok  	github.com/aliyun/terraform-provider-alicloud/alicloud	21.556s

Finish!

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Review by ChenHanZhang — COMMENTED** [review]
approved

**Pull request merged**